### PR TITLE
[pro#547] Make the pro request sidebar sticky

### PR DESCRIPTION
**Pull request by wrightmartin**
* Relevant issue(s)

Fixes https://github.com/mysociety/alaveteli-professional/issues/547

* What does this do?
On desktop this makes the pro request sidebar sticky on scroll 

* Why was this needed?
Part of improvements to the pro request experience https://github.com/mysociety/alaveteli/issues/4505

Precursor to https://github.com/mysociety/alaveteli-professional/issues/548

* Implementation notes
This implementation has limited support - especially for Internet Explorer 11 and below (https://caniuse.com/#feat=css-sticky). Support for older browsers will require a polyfill or JavaScript solution, suggest we discuss this.

### CODE REVIEW

File: app/assets/stylesheets/responsive/_global_layout.scss [hunk 1]
```diff
@@ -140,6 +140,26 @@ textarea{
...
+.sidebar--sticky {  
+  @supports (position: sticky) or (position: -webkit-sticky){
+    $sidebar-vertical-margin: 20px;
+    $adminbar-height: 41px;
+    @include respond-min( $main_menu-mobile_menu_cutoff ){
+      position: -webkit-sticky;
+      position: sticky;
+      top: $sidebar-vertical-margin;
+      //If the sticky sidebar is taller than the screen height, make the contents scrollable
+      max-height: calc(100vh - #{$sidebar-vertical-margin});
```
Comment (garethrees) [inline app/assets/stylesheets/responsive/_global_layout.scss]: I got a scrollbar anyway:

sticky-sidebar-scrollbar

Comment (garethrees) [reply app/assets/stylesheets/responsive/_global_layout.scss]: Just noticed that I can't seem to replicate this with whatdotheyknow-theme, so worth checking what differences there are between the sidebar CSS there and in core.

Comment (wrightmartin) [reply app/assets/stylesheets/responsive/_global_layout.scss]: Fixed

**Comment by wrightmartin** [comment]
It turns out the bug was due to the houdini controls, `.houdini-input` is supposed to be hidden, but it still had a calculated height which, due to the way it was positioned, was throwing the calculation off.

Adding the houdini styles to core fixes this

**Pull request merged**